src/Libraries/PythonNodeModels/PythonEngineSelector .cs

```diff
@@ -1,22 +1,11 @@
-﻿using System;
-using System.Collections.Generic;
+﻿using Dynamo.PythonServices;
+using System;
 using System.Collections.ObjectModel;
 using System.Linq;
 using System.Reflection;
 
 namespace PythonNodeModels
 {
-    /// <summary>
-    /// Enum of possible values of python engine versions.
-    /// TODO: Remove when dynamic loading logic is there then we no longer need a hard copy of the options
-    /// </summary>
-    public enum PythonEngineVersion
```

Comment:
hmm, I think you can safely move this to a new assembly using a type forward if you keep the namespace and typename exactly the same.
https://docs.microsoft.com/en-us/dotnet/standard/assembly/type-forwarding

Reply:
I thought I could forward the type to a new assembly and not have to add a reference to it for all projects that use the type.
This is what I am seeing for a project that uses PythonEngineVersion and only has a reference to PythonNodeModels (the old assembly location)
![image](https://user-images.githubusercontent.com/46732933/140241385-3aa1c01c-b9d0-41d3-9327-68bcb30846f7.png)

Reply:
Maybe I misunderstood the documentation...looks like I need to add a reference to the new assembly location to all the projects that use the PythonEngineVersion type